release/_publishing.sh

```diff
@@ -130,6 +130,11 @@ function upload_release {
 
 	echo "# Uploaded" > ../output.md
 
+	if [ "${LIFERAY_RELEASE_OUTPUT}" = "nightly" ]
+	then
+		ssh root@lrdcom-vm-1 mkdir -p "/www/releases.liferay.com/${LIFERAY_RELEASE_PRODUCT_NAME}/nightly"
```

Comment:
It can stay here, but I think it's unnecessary. That directory exists always, unless someone (our script) erase it.